Mathlib/RingTheory/LaurentSeries.lean

```diff
@@ -454,8 +480,7 @@ theorem single_zpow (n : ℤ) :
       single_inv (n_neg + 1 : ℤ) one_ne_zero, zpow_neg, ← Nat.cast_one, ← Int.ofNat_add,
       Nat.cast_one, inv_inj, zpow_natCast, single_one_eq_pow, inv_one]
 
-instance : Algebra (RatFunc F) F⸨X⸩ :=
-  RingHom.toAlgebra (coeAlgHom F).toRingHom
+instance : Algebra (RatFunc F) F⸨X⸩ := RingHom.toAlgebra (coeAlgHom F).toRingHom
```

Comment:
There was a diamond about this one, right? Or something similar, I don't remember.

Reply:
I was thinking about [this](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/compiling.20behaviour.20within.20one.20file/near/452600793) discussion

Reply:
I forgot that the diamond is now solved, let's forget about it.